cmd/kiam/agent.go

```diff
@@ -45,6 +47,7 @@ func (cmd *agentCommand) Bind(parser parser) {
 
 	parser.Flag("port", "HTTP port").Default("3100").IntVar(&cmd.port)
 	parser.Flag("allow-ip-query", "Allow client IP to be specified with ?ip. Development use only.").Default("false").BoolVar(&cmd.allowIPQuery)
+	parser.Flag("allowed-routes", "Proxy routes matching this regular expression").Default(".*").RegexpVar(&cmd.allowedRoutes)
```

Comment:
Should the flag be plural - `routes` - if we only allow it to be specified once?

Reply:
it's a regular expression which can match many routes

Reply:
Perhaps then we could rename to specify that it's a regexp? So, change it to:

```
kiam server --whitelist-route-regexp=.*
```

Thoughts @sambooo and @ewbankkit? Agree about the plural name though- we don't support multiple regexes so it'd be good to name it singularly to avoid confusion.

Reply:
Naming is the hardest problem 😄.
No particular opinion on this one but I think some documented examples would be good.